src/preface.md

```diff
@@ -1 +1,70 @@
+<p style="text-align:center;"><img src="./assets/esp-logo-black.svg" width="50%"></p>
+
 # Preface
+
+Welcome to a comprehensive guide to using the `esp-rs` ecosystem, created for embedded Rust development on Espressif products. This book serves as a practical resource for developers wishing to utilise the capabilities of Rust in `no_std` environments when working with Espressif hardware platforms.
+
+## Who This Book Is For
+
+This book is intended for people with some experience in Rust and assumes a basic knowledge of embedded development and electronics. The expected target user of our product and reader of this book has a basic understanding of concepts such as cross-compilation, common digital interfaces (UART, SPI, I2C), memory-mapped peripherals and interrupts. For those without prior experience, we recommend first reading the [Scope of This Book][prerequisites] and [Resources][resources] sections to get up to speed.
+
+[prerequisites]: #scope-of-this-book
+[resources]: #additional-resources
+
+## Scope of This Book
+
+In this book, we focus specifically on the `esp-rs` ecosystem. It does **not** serve as:
+
+- A general Rust programming language tutorial: if you are new to Rust, we recommend starting with [The Rust Programming Language][rust-book].
+- An embedded development guide: prior familiarity with basic embedded development is assumed.
+- A collection of examples: This book explains the overall process of working with the `esp-rs` ecosystem. More examples can be found [here][examples] or in the [Awesome ESP Rust][awesome-esp-rust] repository. 
```

Comment:
Maybe rather than linking to the same resources in multiple places, we can just add a sentence after this list telling people to check the Additional Resources section below if they are not familiar with any of these concepts?